.pre-commit-config.yaml

```diff
@@ -7,7 +7,6 @@ repos:
     -   id: end-of-file-fixer
     -   id: check-docstring-first
     -   id: check-json
-  #  -   id: check-added-large-files
```

Comment:
I think this is a good check to include. I commented it out yesterday because it wasn't working, not because I don't like it.